ai_worker/src/storage/storage_manager.py

```
@@ -0,0 +1,295 @@
+                chunks_stored += 1
+
+            return {
+                "file_id": file_meta.file_id,
+                "total_messages": len(messages),
+                "chunks_stored": chunks_stored
+            }
+
+        except Exception as e:
+            # 에러 발생 시 롤백
+            self._rollback_file(file_meta.file_id)
+            raise e
```

Ruff (0.14.5)

Use `raise` without specifying exception name

Remove exception name

(TRY201)